extensions/amp-ad-network-adsense-impl/0.1/test/test-amp-ad-network-adsense-impl.js

```diff
@@ -14,14 +14,31 @@
  * limitations under the License.
  */
 
-import {AmpAdNetworkAdsenseImpl} from '../amp-ad-network-adsense-impl';
+import {
+  AmpAdNetworkAdsenseImpl,
+  resetSharedState,
+} from '../amp-ad-network-adsense-impl';
 import {AmpAdUIHandler} from '../../../amp-ad/0.1/amp-ad-ui'; // eslint-disable-line no-unused-vars
 import {
   AmpAdXOriginIframeHandler,    // eslint-disable-line no-unused-vars
 } from '../../../amp-ad/0.1/amp-ad-xorigin-iframe-handler';
 import {base64UrlDecodeToBytes} from '../../../../src/utils/base64';
 import {utf8Encode} from '../../../../src/utils/bytes';
 import * as sinon from 'sinon';
+import {createIframePromise} from '../../../../testing/iframe';
+import {upgradeOrRegisterElement} from '../../../../src/custom-element';
+
+
+function getAdsenseImplElement(attributes, opt_doc, opt_tag) {
```

Comment:
Rename to createAdsenseImplElement given this actually changes the state of the document